vignettes/betareg.Rmd

```diff
@@ -0,0 +1,195 @@
+round(coef(fit2), 2)
+```
+
+``` {r simulated-fit-print, echo=FALSE}
+round(coef(fit1), 2)
+round(coef(fit2), 2)
+```
+
+The usual posterior analyses are available in **rstanarm**.
+The plots below illustrate simulated values of the outcome variable. The incorrect
+model noticeably fails to capture the top of the distribution consistently in comparison
+to the true model.
+```{r simulated-analysis, fig.height=5}
+library(gridExtra)
+plot1 <- pp_check(fit1) + ggtitle("True Model: y ~ x | z")
+plot2 <- pp_check(fit2) + ggtitle("False Model: y ~ x")
+grid.arrange(plot1, plot2, ncol = 2)
+```
+
+We can also compare models by evaluating the expected log pointwise predictive density (`elpd`), which can be calculated
+using the `loo` package.
+
+``` {r simulated-loo}
+library(loo)
+loo1 <- loo(fit1)
+loo2 <- loo(fit2)
+compare(loo1, loo2)
+```
+
+The difference in `elpd` is negative indicating that the expected predictive accuracy for
+the first model is higher.
+
+# An Example Using Real Data
+In some applied contexts it may be necessary to work with an outcome variable that is a proportion. If the proportion
+is bound on the open unit interval then beta regression can be considered a reasonable estimation method. The
+`betareg` package provides a dataset on the proportion of crude oil converted to gasoline after distillation
+and fractionation. This variable is defined as yield. Below `stan_betareg` is used to model yield as a function of temperature,
+pressure, and the batch of conditions. 
+
+```{r, gas-fit, results="hide"}
+library(rstanarm)
+data("GasolineYield", package = "betareg")
+gas_fit1 <- stan_betareg(yield ~ temp + batch, data = GasolineYield, link = "logit", algorithm = "sampling")
+gas_fit2 <- stan_betareg(yield ~ temp | pressure, data = GasolineYield, link = "logit", algorithm = "sampling")
+round(coef(gas_fit1), 2)
+round(coef(gas_fit2), 2)
+```
+
+``` {r, gas-print, echo=FALSE}
+round(coef(gas_fit1), 2)
+round(coef(gas_fit2), 2)
+```
+
+The plots below illustrate simulated values of gasoline yield.
+While the first model accounts for variation in batch conditions its predictions
+looks somewhat uniform rather than resembling the peaked and skewed behavior of the true data. The second
+model does a relatively better job at capturing the shape of the distribution, however its location is off
+as it is centered around 0.50 rather than 0.20.
+
+```{r gas-analysis, fig.height=5}
+library(gridExtra)
+gas_plot1 <- pp_check(gas_fit1) + ggtitle("gas_fit1")
+gas_plot2 <- pp_check(gas_fit2) + ggtitle("gas_fit2")
+grid.arrange(gas_plot1, gas_plot2, ncol = 2)
+```
+``` {r, gas-loo}
+library(loo)
+gas_loo1 <- loo(gas_fit1)
+gas_loo2 <- loo(gas_fit2)
+compare(gas_loo1, gas_loo2)
+```
+
+Additionally, evaluating the expected log predictive distribution using `loo`
+reveals that the second of the two models is preferred.
+
+Note that the `sqrt` link function (applied to the second set of regressors) is known to be unstable, so it
```

Comment:
Unless you're using `sqrt` link in the vignette I think you can get rid of this given that it's now in the function doc.